source/change-streams/change-streams.rst

```diff
@@ -575,6 +548,21 @@ A possible interface for this callback MAY look like:
 
 This MUST NOT be implemented in synchronous drivers. This MAY be implemented in asynchronous drivers.
 
+Updating the Cached Resume Token
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following rules describe how to update the cached ``resumeToken``:
```

Comment:
Markup:
![image](https://user-images.githubusercontent.com/44870738/56134729-d9c86480-5f97-11e9-89de-7db0b7c0046f.png)

Reply:
Looks like this has been fixed since this comment was posted.